cfg/Instructions.h

```diff
@@ -47,6 +47,7 @@ enum class Tag : uint8_t {
     YieldLoadArg,
     Cast,
     TAbsurd,
+    VolatileRead,
```

Comment:
I don't know what other IRs call similar instructions, but what do you think about `KeepAlive` or something like that?

Reply:
I was going on `volatile` from C/C++ (particularly as it applies to inline assembly), which to a first approximation means "you can't assume anything about what happens with this", but `KeepAlive` wfm.